### ISSUE TITLE
Buying new pots! (Has Britany's email for discount)

### ISSUE THREAD

**Issue by christophe-rd** [christophe-rd/fuelinex#19]
@lizzieinvancouver I aim to repot the trees in the winter. From the few individuals I checked, the roots don't seem to have been much limited by the pots, so that's great! 
All trees are currently in 2-gal pots. I initially thought of putting them in 4-gal pots, but it turns out that size doesn't really exist. Therefore I have to choose between 3 or 5 gallons. I went to check them at a store in Burnaby yesterday and I think 5 gal is way to big because 1. I don't think the trees would fit in the greenhouse anymore, 2. it would take significantly longer to repot them and 3. we would have to buy a lot more soil. Therefore 3-gallon pots would be fine in my opinion. 
They are +- 3$ each and I have 591 trees left in the experiment. They come in packs of 25, so it would be +-1800$. What do you think?
I am waiting from a quote from them to have the exact price.

**Comment by christophe-rd** [comment]
Also, I think we could argue that this is equipment since they could be used for multiple experiments.

**Comment by christophe-rd** [comment]
@lizzieinvancouver I got the quote from Jon's plants. [You can find it here](https://github.com/christophe-rd/fuelinex/blob/49e8262f295ba8e7616d6518ae24f9cf90ebf5ce/notes/plant_pots/quote_3-gal_pots.pdf)
Could you please let me know your thoughts? There is no rush though.

**Comment by lizzieinvancouver** [comment]
@christophe-rd We can order these as equipment, but we should get into the habit of asking for a discount. @buniwuuu got a great one last spring from Hoskin. Can you get the email she used and paste it here so you, me and @wangxm-forest can use it? Thanks.

**Comment by lizzieinvancouver** [comment]
@christophe-rd Oh, and then adjust the email for this and ask for a discount.

**Comment by christophe-rd** [comment]
@lizzieinvancouver this is the quote with the discounted price. I re-asked for it and this is what I got. They will also charge either 100 or 150$ for delivery (they need to figure out if the company they use deliver to UBC). 
I asked another company for a quote and they can't get an order this big until spring. They referred me to Jon's plants.

Do you want me yo re-email them anyway?

**Comment by lizzieinvancouver** [comment]
@christophe-rd Thanks, I cannot see the quote, however. Did you mean to link to it somehow? Can you also share the email that @buniwuuu used for Hoskin? Thanks.

**Comment by buniwuuu** [comment]
Hi @lizzieinvancouver @christophe-rd ! I submitted my inquiry through their website and bjones@hoskin.ca was the person that reached out to me after!

**Comment by lizzieinvancouver** [comment]
@buniwuuu Thanks, but we actually wanted the text of the email you used to get the discount.

**Comment by buniwuuu** [comment]
Oh! Sorry about that! Here it is:

I hope this message finds you well. My name is Britany Wu, and I am a
member of the Temporal Ecology Lab led by Dr. Elizabeth Wolkovich at
UBC. I am writing to inquire if you have any promotions for CFI
purchases and if I can request a quote for 80 Onset HOBO UA-002-64
Pendant Temperature/Light Data Logger, 64 KB.



Thank you in advance and I look forward to hearing from you!

**Comment by lizzieinvancouver** [comment]
@buniwuu Thanks, but I am looking for the LAST EMAIL you send right before we finalized the order. I think this may be the first one? I asked you to press them for a better discount and you said you did with help from chatGPT. I am looking for *that* particular email if you have it. Thanks.

**Comment by lizzieinvancouver** [comment]
@buniwuu Thanks, but I am looking for the LAST EMAIL you send right before we finalized the order. I think this may be the first one? I asked you to press them for a better discount and you said you did with help from chatGPT. I am looking for *that* particular email if you have it. Thanks.

**Comment by lizzieinvancouver** [comment]
@buniwuu Thanks, but I am looking for the LAST EMAIL you send right before we finalized the order. I think this may be the first one? I asked you to press them for a better discount and you said you did with help from chatGPT. I am looking for *that* particular email if you have it. Thanks.

**Comment by buniwuuu** [comment]
@lizzieinvancouver ahh I see. Here it is!

Hi Brad,

Thank you for getting back to me so quickly. After further consideration, we decided to expand our order to include an additional 10 units of MX2202, bringing our total to 90 MX2202 units, along with 18 packs of mounting boots and 20 units of MX2301A.

As we're in the process of comparing prices from different suppliers and would love to prioritize Canadian businesses, we're hoping to inquire about the possibility of receiving an additional discount. Specifically, we're wondering if there's room for a 23% CFI discount on our total order.

Thank you for your assistance and I look forward to hearing from you!

**Comment by lizzieinvancouver** [comment]
@buniwuuu That's great -- thank you! @christophe-rd and @wangxm-forest please use this template to request a discount once you have honed in on what you want to order from a supplier.

**Comment by lizzieinvancouver** [comment]
@christophe-rd I don't think you need to try another supplier given the last one did not have the pots currently, but can you tell me the discount? The link leads to a quote of $1740, which is identical to the first quote.

**Comment by christophe-rd** [comment]
@lizzieinvancouver sorry for the misunderstanding! They didn't submit the quote with no discount. I'll ask for it now.
